src/tests/ngraph_helpers/snippets_ngraph_functions/src/precision_propagation_function.cpp

```diff
@@ -19,7 +19,8 @@ std::shared_ptr<ngraph::Function> PrecisionPropagationAddFunction::get(
     const std::pair<element::Type, element::Type>& convertion_before_op1,
     const element::Type convertion_before_op2_1,
     const std::pair<element::Type, element::Type>& convertion_before_op2_2,
-    const element::Type convertion_after_op2) {
+    const element::Type convertion_after_op2,
+    const element::Type convertion_before_result) {
```

Comment:
Why do you pass `PartialShape` and `std::pair<element::Type, element::Type>` by const reference, but `element::Type` by value? Shoudn't it be const reference as well?

Reply:
should not. argument has default value

Reply:
refactored, done